### PR TITLE
minor changes to config files

### PR DESCRIPTION
**Pull request by ctf0**
1- changed some security settings to make it more secure out-of-the-box
2- changed the data-upload limit for something more reasonable
3- added an env key for `SESSION_DOMAIN` for easier user exp
4- fixed some docs typos and made config files more consistent
5- added `PATCH` verb to **csrf**

### CODE REVIEW

**Comment by RomainLanz** [comment]
Nice one !

**Review by RomainLanz — APPROVED** [review]

**Comment by ctf0** [comment]
i will update the docs once the PR is merged.

File: config/bodyParser.js
Comment (thetutlage) [inline config/bodyParser.js]: 125MB for post data is too much. This does not include file uploads and should be under `20MB`. If someone has requirements they should come and toggle this value, atleast with that they would know what they are doing.

File: .env.example
Comment (RomainLanz) [inline .env.example]: We should keep sqlite

File: .env.example
Comment (RomainLanz) [inline .env.example]: This is the default port

File: config/database.js
Comment (RomainLanz) [inline config/database.js:21]: Why?

File: config/database.js
Comment (RomainLanz) [inline config/database.js:47]: Why?

File: config/database.js
Comment (RomainLanz) [inline config/database.js:51]: It's the same as mysql

File: config/database.js
Comment (RomainLanz) [inline config/database.js:67]: Why?

**Review by RomainLanz — CHANGES_REQUESTED** [review]

File: .env.example
Comment (ctf0) [reply .env.example]: the reason i changed because most of ppl use mysql and to also keep the db env keys unless the user wants to use sqlite then he can remove them

File: .env.example
Comment (ctf0) [reply .env.example]: yes but sometimes the user use a different one, so its kept just incase

File: config/database.js
Comment (ctf0) [reply config/database.js:21]: because from the docs, some dbs have more than one driver, so its left for the user to decide which driver he wants to install/use

File: config/database.js
Comment (ctf0) [reply config/database.js:51]: i know, but i added as it was mentioned in the docs, so i thought maybe there is something different

File: .env.example
Comment (RomainLanz) [reply .env.example]: SQLite driver can works without installing anything (only the sqlite package). That's why it is use by default.

File: bootstrap/app.js
Comment (RomainLanz) [inline bootstrap/app.js]: This is an example command, it should stick to the top to be removed easily.

File: .env.example
Comment (ctf0) [reply .env.example]: no problem, we can change it back then, but in that case the other keys for db in env is useless, what if i removed it aswell and in case user needs a different driver then he can add what he wants ?

**Review by RomainLanz — CHANGES_REQUESTED** [review]

File: bootstrap/app.js
Comment (ctf0) [reply bootstrap/app.js]: will keep it along with greet at the top then

**Comment by RomainLanz** [comment]
I'll let @thetutlage review it and react to changes.

**Comment by ctf0** [comment]
@RomainLanz btw,if i've a couple of ideas what is the best place to put ? i dont think issuing a ticket on the main repo is the correct action

**Comment by RomainLanz** [comment]
You should use the https://github.com/adonisjs/discussion repository.

**Review by thetutlage — COMMENTED** [review]
@ctf0 Their are some reviews, please look into them so that I can merge the PR

File: config/bodyParser.js
Comment (ctf0) [reply config/bodyParser.js]: okay no problem

File: .env.example
Comment (thetutlage) [inline .env.example:8]: Session cookie does not needs to be here. Neither the session domain. 

I will never change the `cookie` name for each environment. Also their should be `DOMAIN` variable, which can be used by application internally and should not be part of core

Comment (ctf0) [reply .env.example:8]: the reason behind `SESSION_COOKIE & SESSION_DOMAIN` is to minimize being effected by any bug that could be used by an attacker which would effect all the apps made by adonis

**Comment by thetutlage** [comment]
This PR has lot of stuff going. I would suggest to split into multiple PR's

**Pull request closed without merging**
